src/ifk_scb_compilations/objects/emissions_kommun.py

```diff
@@ -0,0 +1,98 @@
+"""Fetch emission data by kommun.
+
+Observera att i tabellen över utsläpp från transporter ingår för närvarande endast vägtransporter.
+Miljöräkenskapernas statistik om utsläpp till luft utgår från ett produktionsperspektiv och redovisar därför
+direkta utsläpp från svenska ekonomiska aktörer. Produktionsperspektivet är annorlunda från det territoriella
+perspektivet (som används för att rapportera Sveriges utsläpp till FN), vilket avgränsas till utsläpp som
+sker inom Sveriges gränser. Samtidigt används ett territoriellt perspektiv i mycket av den underlagsdata
+som används för att producera miljöräkenskapernas statistik om utsläpp till luft. För att justera för
+skillnaden mellan miljöräkenskapernas produktionsperspektiv och det territoriella perspektivet används
+en så kallad residensjustering. Residensjusteringen appliceras framförallt på transportrelaterade utsläpp.
+Mer information om beräkningsmetodiken och underlagsdata som används för residensjusteringen redovisas i
+statistikens Kvalitetsdeklarationen.
+"""
+
+
+import pandas as pd
+from pyscbwrapper import SCB
+
+
+class FetchData:
```

Comment:
We (@docNord) had a discussion about calling methods `fetch` or `get`? We went with `get` for some reason. Could be nice to be consistent across repos.

Reply:
get applied.